### ISSUE TITLE
Some text and icons in manage resource model too close

### ISSUE THREAD

**Issue by SDScandrettKint** [archesproject/arches#10911]
Some of the text and icons in the Resource Model Manage menu are too close e.g. Clone model, Export model and Delete model:
![image](https://github.com/archesproject/arches/assets/119508494/e61562cc-635b-4ca2-a653-807ec22cadcd)

**Comment by samuelscandrett** [comment]
Reopened, as still an issue in 7.6.x despite [PR](https://github.com/archesproject/arches/pull/10912) - I wonder whether it was reset by linting or pre-commit styling?

@jacobtylerwalls shall I create another PR same as before and target 7.6?

**Comment by jacobtylerwalls** [comment]
>I wonder whether it was reset by linting or pre-commit styling?

Looks like it regressed in #10893.

> shall I create another PR same as before and target 7.6?

Sure, and two PRs, one for 7.6 and one for 8 would be great since it will get out in front of the merge conflict.

**Comment by jacobtylerwalls** [comment]
I wonder if you might look into a solution other than using empty text for padding? That's probably why it looked like it could be removed.

**Comment by samuelscandrett** [comment]
I agree there should be a better solution than whitespace for padding, as this can get easily removed without realising it is important.

All of the items in the manage list use whitespace, for example the space after /i> and {% trans, they were just not affected by the previous commit that caused the regression.

https://github.com/archesproject/arches/blob/429603e3a9e4a0fc8d7d0732fcd619c97eee0217/arches/app/templates/navbar/graph-designer-menu.htm#L15C87-L15C88

<br>
As a potential solution, we could use a non-breaking space "nbsp" between the icon and text?  This is used in the example "Fixed Width Icons" on https://fontawesome.com/v4/examples/.

This would look as such:
```
<div class="menu-item-title"><i class="fa fa-upload"></i>&nbsp{% trans "Import Model" %}</div>

...

<div class="menu-item-title"><i class="fa fa-clone"></i>&nbsp;<span data-bind="text: graph.isresource() ? {% quoted_trans _("Clone Model") %} : {% quoted_trans _("Clone Branch") %} "></span></div>
```

Though for consistency we should remove all spaces before and after the closing icon and `{% trans` or `<span` to not introduce an extra space.
